lib/src/components/TextInputs/TextInput.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import { StyleSheet, TextInput as RNTextInput, TextInputProps } from 'react-native'
+import { Colors, Fonts } from '../../res/'
+
+interface Props extends TextInputProps {}
+
+interface State {
+  focused: boolean
+}
+
+export default class TextInput extends Component<Props, State> {
+  constructor(props: Props, state: State) {
+    super(props, state)
+    this.state = {
+      focused: false,
+    }
+  }
+
+  public render() {
+    const borderStyle = this.state.focused
+      ? { borderColor: Colors.AlgaeGreen }
+      : { borderColor: Colors.White }
+    return (
+      <RNTextInput
+        {...this.props}
+        style={[styles.textInput, this.props.style, borderStyle]}
+        onFocus={this.onFocus.bind(this)}
+        onBlur={this.onBlur.bind(this)}
+      />
+    )
+  }
+
+  public onFocus() {
+    this.setState(_ => ({
+      focused: true,
+    }))
+  }
+
+  public onBlur() {
+    this.setState(_ => ({
+      focused: false,
+    }))
+  }
+}
+
+const styles = StyleSheet.create({
+  textInput: {
+    height: 52,
```

Comment:
How about start putting these values in a dimens file?

Reply:
Yeah we can do that. I just don't see a reason for a `dimens` file yet. For fonts, colors, icons and strings I totally see it.

Maybe because this value would probably never be reused and would be a 1-1 relation most of the time. Unless we have a very strict design system.

I would leave this decision for the future.

Reply:
👍